2016-06-10 11:20:43.788 -0600	START		Running CalculatorTesting.feature
2016-06-10 11:20:44.914 -0600	imagefound	Calculator	found at (613, 223)	2	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 11:20:45.003 -0600	click		at (613, 223)	6	on_that_the_calculator_app_is_running_and_in_focus	CalculatorTesting.script<SOURCE>
2016-06-10 11:20:45.004 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 11:20:45.004 -0600	LogSuccess		that the Calculator app is running and in focus
2016-06-10 11:20:46.249 -0600	click	CalcClearButton	at (677, 367)	2	on_there_is_no_current_value_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 11:20:46.251 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 11:20:46.251 -0600	LogSuccess		there is no current value in the calculator
2016-06-10 11:20:46.977 -0600	typetext		4[return]	3	on_i_enter_num___num_in_the_calculator	CalculatorTesting.script<SOURCE>
2016-06-10 11:20:46.977 -0600	LogSuccess		I enter 2 + 2 in the calculator
2016-06-10 11:20:46.978 -0600	LogSuccess		I enter 2 + 2 in the calculator
2016-06-10 11:20:46.979 -0600	LogWarning		the value returned should be 4
2016-06-10 11:20:46.980 -0600	EndTestCase		(Duration:"3.189737", Errors:"0", Exceptions:"0", StartTime:"2016-06-10 11:20:43 -0600", Successes:"6", TestCase:"CalculatorTesting.feature", Warnings:"1")
2016-06-10 11:20:48.110 -0600	FAILURE	Screen_Error.png	1 scenarios (1 failed, 0 passed) 4 steps (0 failed, 1 skipped, 3 passed) Execution Time 0:00:03
